licenses/CDDL-1.1

```diff
@@ -0,0 +1,363 @@
+          Participant is a Contributor or the Original Software where the
+          Participant is the Initial Developer) directly or indirectly
+          infringes any patent, then any and all rights granted directly or
+          indirectly to You by such Participant, the Initial Developer (if
+          the Initial Developer is not the Participant) and all Contributors
+          under Sections 2.1 and/or 2.2 of this License shall, upon 60 days
+          notice from Participant terminate prospectively and automatically
+          at the expiration of such 60 day notice period, unless if within
+          such 60 day period You withdraw Your claim with respect to the
+          Participant Software against such Participant either unilaterally
+          or pursuant to a written agreement with Participant.
+
+     6.3. If You assert a patent infringement claim against Participant
+          alleging that the Participant Software directly or indirectly
+          infringes any patent where such claim is resolved (such as by
+          license or settlement) prior to the initiation of patent
+          infringement litigation, then the reasonable value of the
+          licenses granted by such Participant under Sections 2.1 or
+          2.2 shall be taken into account in determining the amount or
+          value of any payment or license.
+
+     6.4. In the event of termination under Sections 6.1 or 6.2 above,
+          all end user licenses that have been validly granted by You or
+          any distributor hereunder prior to termination (excluding
+          licenses granted to You by any distributor) shall survive
+          termination.
+
+7. LIMITATION OF LIABILITY.
+
+     UNDER NO CIRCUMSTANCES AND UNDER NO LEGAL THEORY, WHETHER TORT (INCLUDING
+     NEGLIGENCE), CONTRACT, OR OTHERWISE, SHALL YOU, THE INITIAL DEVELOPER,
+     ANY OTHER CONTRIBUTOR, OR ANY DISTRIBUTOR OF COVERED SOFTWARE, OR ANY
+     SUPPLIER OF ANY OF SUCH PARTIES, BE LIABLE TO ANY PERSON FOR ANY INDIRECT,
+     SPECIAL, INCIDENTAL, OR CONSEQUENTIAL DAMAGES OF ANY CHARACTER INCLUDING,
+     WITHOUT LIMITATION, DAMAGES FOR LOSS OF GOODWILL, WORK STOPPAGE, COMPUTER
+     FAILURE OR MALFUNCTION, OR ANY AND ALL OTHER COMMERCIAL DAMAGES OR LOSSES,
+     EVEN IF SUCH PARTY SHALL HAVE BEEN INFORMED OF THE POSSIBILITY OF SUCH
+     DAMAGES. THIS LIMITATION OF LIABILITY SHALL NOT APPLY TO LIABILITY FOR
+     DEATH OR PERSONAL INJURY RESULTING FROM SUCH PARTY'S NEGLIGENCE TO THE
+     EXTENT APPLICABLE LAW PROHIBITS SUCH LIMITATION. SOME JURISDICTIONS DO NOT
+     ALLOW THE EXCLUSION OR LIMITATION OF INCIDENTAL OR CONSEQUENTIAL DAMAGES,
+     SO THIS EXCLUSION AND LIMITATION MAY NOT APPLY TO YOU.
+
+8. U.S. GOVERNMENT END USERS.
+
+     The Covered Software is a "commercial item," as that term is defined in
+     48 C.F.R. 2.101 (Oct. 1995), consisting of "commercial computer software"
+     (as that term is defined at 48 C.F.R. ? 252.227-7014(a)(1)) and
```

Comment:
Broken encoding? This should be a section sign "§" not a question mark.
Where did you take this license file from?

Reply:
I've rebased this feature branch to the current master, and dropped its CDDL-1.1 license file in favor of the ones in master (from commit https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=9a8505e9128e8e80dc11b330e1b795ca465b9061)